contrib/nydusify/pkg/committer/commiter.go

```diff
@@ -845,3 +846,89 @@ func (cm *Committer) obtainBootStrapInfo(ctx context.Context, BootstrapName stri
 	}
 	return output.FsVersion, strings.ToLower(output.Compressor), nil
 }
+
+// Constants for container operations
+const (
+	// DefaultNamespace is the default containerd namespace
+	DefaultNamespace = "default"
+	// MaxContainerIDLength is the maximum length of a container ID
+	MaxContainerIDLength = 64
+)
+
+// Initialize initializes the committer with the given options
+func (cm *Committer) Initialize(ctx context.Context, opt *Opt) error {
```

Comment:
Maybe we can refer 
 the nerdctl implementation about container walker: https://github.com/containerd/nerdctl/blob/31b4e49db76382567eea223a7e8562e0213ef05f/pkg/idutil/containerwalker/containerwalker.go#L53